test/unit/orchestration/dhcp_test.rb

```diff
@@ -154,6 +154,9 @@ def test_bmc_should_have_valid_dhcp_record
     bmc = h.interfaces.bmc.first
     assert !bmc.new_record?
     bmc.mac = bmc.mac.succ
+    binding.pry
```

Comment:
not on Jenkins, please :)  (though pry shouldn't be installed, so it may error here first)